AutomatedTesting/Gem/PythonTests/largeworlds/CMakeLists.txt

```diff
@@ -143,7 +144,20 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS AND PAL_TRAIT_
         NAME AutomatedTesting::LandscapeCanvasTests_Periodic
         TEST_SERIAL
         TEST_SUITE periodic
-        PATH ${CMAKE_CURRENT_LIST_DIR}/landscape_canvas/test_LandscapeCanvas_Periodic.py
+        PATH ${CMAKE_CURRENT_LIST_DIR}/landscape_canvas/TestSuite_Periodic.py
+        RUNTIME_DEPENDENCIES
+           AZ::AssetProcessor
+           Legacy::Editor
+           AutomatedTesting.Assets
+        COMPONENT
+            LargeWorlds
+    )
+
+    ly_add_pytest(
+        NAME AutomatedTesting::LandscapeCanvasTests_Main_Optimized
```

Comment:
It may be a good idea to merge all the LargeWorlds optimized testsuite into a single one. This way all the tests will be parallelized together

Reply:
I disagree. If we really need to shave that much time off, I can, but it'll make investigating/organizing things a lot more difficult. Large Worlds is not a single feature.

Reply:
Physics is not a single feature either, but its compromised of other subfeatures (like ragdolls, force regions, joints, etc) and we still have a single test suite that compromises every part.
The benefit I can see of having them separate is is that its easier to tell what part has broken, but its still possible by to tell by looking at the results. For the optimizations I believe that it would definitely make a difference, not saying that is required for this PR though

Reply:
That's fair, will look at this for future improvements. Thanks!